packages/p2p/src/components/orders/chat/chat.jsx

```diff
@@ -13,14 +13,6 @@ import 'Components/orders/chat/chat.scss';
 const Chat = observer(() => {
     const { sendbird_store } = useStores();
 
-    if (sendbird_store.is_chat_loading) {
-        return (
-            <div className='order-chat'>
-                <Loading is_fullscreen={false} />;
-            </div>
-        );
-    }
-
```

Comment:
Is this causing the TrackJS issue? This loader is necessary so let's keep it but try to improve it.